docs/content/releases.md

```diff
@@ -7,6 +7,46 @@ toc: true
 docs: "DOCS-616"
 ---
 
+## NGINX Ingress Controller 3.2.0
+
+27 June 2023
+
+FEATURES:
+
+* [3790](https://github.com/nginxinc/kubernetes-ingress/pull/3790) Gunzip for VS
+* [3863](https://github.com/nginxinc/kubernetes-ingress/pull/3863) OIDC - relaxed OIDC scope validation
+* [3925](https://github.com/nginxinc/kubernetes-ingress/pull/3925) Specify runAsNonRoot in daemon-set manifests. Thanks to [Valters Jansons](https://github.com/sigv).
+* [4001](https://github.com/nginxinc/kubernetes-ingress/pull/4001) Add support for the SameSite sticky cookie attribute.
+* [4031](https://github.com/nginxinc/kubernetes-ingress/pull/4031) Add Alpine image with FIPS inside.
+
```

Comment:
We should add a line about license tracking, even though it is delivered separate.  
The linked PR can be the doc.
"Added utilization tracking for supported (paid) customers"